ci/build_wheel_cugraph-dgl.sh

```diff
@@ -3,7 +3,11 @@
 
 set -euo pipefail
 
-package_dir="python/cugraph-dgl"
+if [ "$RAPIDS_PY_VERSION" != "3.13" ]; then
```

Comment:
Shouldn't this be unreachable, because we skip the wheel build job for Python 3.13?

We also skip ARM but we don't have exceptions in the script for that. I would favor just leaving this script as-is and letting the filtering be done at the GHA workflow level.

The same applies to all bash scripts in this PR.